pybossa/util.py

```diff
@@ -1108,9 +1110,10 @@ def sign_task(task):
         task['signature'] = signature
 
 
-def get_now_plus_delta_ts(**kwargs):
-    return (datetime.utcnow() + timedelta(**kwargs))
-
+def get_time_plus_delta_ts(time, **kwargs):
+    if isinstance(time, string_types):
```

Comment:
'six' module is for Python 2 and 3 compatibility. We can use 'str' instead of 'string_types'

Reply:
sure, will update